kernel/packages/shared/social/hotScenes.ts

```diff
@@ -30,6 +37,12 @@ type CandidateCrowdedScene = {
   usersCount: number
 }
 
+export type HotSceneInfo = {
+  baseCoords: { x: number; y: number }
+  usersTotalCount: number
```

Comment:
maybe it's more useful to have a collection of the user-IDs/addressed that are inside the scene?

Reply:
> maybe it's more useful to have a collection of the user-IDs/addressed that are inside the scene?
> (we can later calculate/take the count() on that collection)

Wouldn't be better to have a separate api to get all connected players and their location?
Anyway we can also add that when we need it, do we need that now?

Reply:
nah, it's fine.